src/number/complete.rs

```diff
@@ -1357,29 +1358,29 @@ where
 ///
 /// assert_eq!(parser(&b"01AE"[..]), Ok((&b""[..], 0x01AE)));
 /// assert_eq!(parser(&b"abc"[..]), Ok((&b""[..], 0x0ABC)));
-/// assert_eq!(parser(&b"ggg"[..]), Err(Err::Error((&b"ggg"[..], ErrorKind::IsA))));
+/// assert_eq!(parser(&b"ggg"[..]), Err(Err::Error((&b"ggg"[..], ErrorKind::OneOf))));
```

Comment:
breaking change ?

Reply:
Yes, though in this case it does seem like the internal implementation's error type *should* be an implementation detail. Neither `IsA` or `OneOf` seem like the appropriate contract. What's the library's view on breaking changes? Do them and document? We could do some sort of error conversion from to `ErrorKind::HexU32` or something else.